uportal-war/src/main/webapp/media/skins/respondr/defaultSkin/less/variables.less

```diff
@@ -81,8 +81,8 @@
  * 2) The value must be on the same line as the variable name and be terminated with a semicolon due to
  *    regex searches by skin manager.
  */
-@color3: #317ab9;  // light muted blue
-@color3-light: lighten(@color3, 15%);
+@color3: #005aA9;  // light muted blue
+@color3-dark: darken(@color3, 15%);
```

Comment:
:warning: Avoid deleting `color3-light`, this may be dependend on by custom skins.
:white_check_mark: Instead keep `color3-light` and add `color3-dark` in addition.